Formula/kde-kapidox.rb

```diff
@@ -0,0 +1,65 @@
+class KdeKapidox < Formula
+  desc "Frameworks API Documentation Tools"
+  homepage "https://api.kde.org/frameworks/kapidox/html/index.html"
+  url "https://download.kde.org/stable/frameworks/5.79/kapidox-5.79.0.tar.xz"
+  sha256 "2752c83d9b72245c94d000b5568a457817dc5b6bfcd550daa5f046d5ba9e6462"
+  head "https://invent.kde.org/frameworks/kapidox.git"
+
+  depends_on "cmake" => [:build, :test]
+  depends_on "kde-extra-cmake-modules" => [:build, :test]
+  depends_on "ninja" => :build
+  depends_on "python@3.9" => :build
+  depends_on "qt@5" => :build
+
+  resource "Jinja2" do
+    url "https://files.pythonhosted.org/packages/64/a7/45e11eebf2f15bf987c3bc11d37dcc838d9dc81250e67e4c5968f6008b6c/Jinja2-2.11.2.tar.gz"
+    sha256 "89aab215427ef59c34ad58735269eb58b1a5808103067f7bb9d5836c651b3bb0"
+  end
+
+  resource "MarkupSafe" do
+    url "https://files.pythonhosted.org/packages/b9/2e/64db92e53b86efccfaea71321f597fa2e1b2bd3853d8ce658568f7a13094/MarkupSafe-1.1.1.tar.gz"
+    sha256 "29872e92839765e546828bb7754a68c418d927cd064fd4708fab9fe9c8bb116b"
+  end
+
+  resource "PyYAML" do
+    url "https://files.pythonhosted.org/packages/64/c2/b80047c7ac2478f9501676c988a5411ed5572f35d1beff9cae07d321512c/PyYAML-5.3.1.tar.gz"
+    sha256 "b8eac752c5e14d3eca0e6dd9199cd627518cb5ec06add0de9d32baeee6fe645d"
+  end
+
+  def cmake_args
+    %W[
+      -DCMAKE_INSTALL_PREFIX=#{libexec}
+      -DCMAKE_BUILD_TYPE=Release
+      -DCMAKE_FIND_FRAMEWORK=LAST
+      -DCMAKE_VERBOSE_MAKEFILE=ON
+      -Wno-dev
+    ]
+  end
+
+  def install
+    args = cmake_args
+    args << "-DPYTHON_EXECUTABLE=" + Formula["python"].bin + "/python3"
+
+    mkdir "build" do
+      system "cmake", "..", "-G", "Ninja", *args
+      system "ninja"
+      system "ninja", "install"
+      prefix.install "install_manifest.txt"
+    end
+
+    xy = Language::Python.major_minor_version "python3"
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python#{xy}/site-packages"
+    resources.each do |r|
+      r.stage do
+        system "python3", *Language::Python.setup_install_args(libexec/"vendor")
+      end
+    end
+
+    bin.install Dir[libexec/"bin/*"]
+    bin.env_script_all_files(libexec/"bin", PYTHONPATH: ENV["PYTHONPATH"])
+  end
+
+  test do
+    assert_predicate bin/"kapidox_generate", :exist?
```

Comment:
Could you propose a test that exercises some functionality of this formula?

Reply:
@yurikoles are you familiar with kapidox? I am getting the following:
```
~ /opt/homebrew/Cellar/kde-kapidox/5.79.0/bin/kapidox_generate
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/kde-kapidox/5.79.0/libexec/bin/kapidox_generate", line 4, in <module>
    __import__('pkg_resources').run_script('kapidox==5.79.0', 'kapidox_generate')
  File "/opt/homebrew/lib/python3.9/site-packages/pkg_resources/__init__.py", line 3243, in <module>
    def _initialize_master_working_set():
  File "/opt/homebrew/lib/python3.9/site-packages/pkg_resources/__init__.py", line 3226, in _call_aside
    f(*args, **kwargs)
  File "/opt/homebrew/lib/python3.9/site-packages/pkg_resources/__init__.py", line 3255, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/opt/homebrew/lib/python3.9/site-packages/pkg_resources/__init__.py", line 568, in _build_master
    ws.require(__requires__)
  File "/opt/homebrew/lib/python3.9/site-packages/pkg_resources/__init__.py", line 886, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/opt/homebrew/lib/python3.9/site-packages/pkg_resources/__init__.py", line 772, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'kapidox==5.79.0' distribution was not found and is required by the application
```

I checked `do-caveats.sh` but don't see any hints.

Reply:
@carlocab I removed it for now from this batch